### PR TITLE
Upgrade libs

### PR DESCRIPTION
**Pull request by carlonzo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,8 +2,9 @@ language: android
 
 android:
   components:
-    - build-tools-21.1.1
-    - android-18
+    - tools
```
Comment (carlonzo) [inline .travis.yml]: needed to use build-tools-23.0.2

**Comment by carlonzo** [comment]
there are failing tests, but it is not my fault! check my previous PRs to see how I've solved them

**Comment by orhanobut** [comment]
Hi, Thanks for the pull requests. I'm back now I'll check them on weekends.

**Comment by orhanobut** [comment]
I'll merge this one.

**Pull request merged**